operator/README.md

```diff
@@ -0,0 +1,16 @@
+# Console Operator
+
+This directory provides files to install the Console Operator. To install, use the kustomization.yaml file:
```

Comment:
Do you want to clarify that this just installs the operator?  Do you want to describe (or just provide a link to) how to use this to actually install _console_?